Converter_ACDC.pretty/Converter_ACDC_Recom_RAC20-xxDK_THT.kicad_mod

```diff
@@ -0,0 +1,41 @@
+(module Converter_ACDC_Recom_RAC20-xxDK_THT (layer F.Cu) (tedit 5F53E26B)
+  (descr "Recom RAC20-xxDK https://recom-power.com/pdf/Powerline_AC-DC/RAC20-K.pdf#page=6 Rev. 6/2020 (Script generated with StandardBox.py) (Recom RAC20-xxDK https://recom-power.com/pdf/Powerline_AC-DC/RAC20-K.pdf#page=6 Rev. 6/2020)")
+  (tags "Recom RAC20-xxDK")
+  (fp_text reference REF** (at -1 -4.59) (layer F.SilkS)
+    (effects (font (size 1 1) (thickness 0.15)))
+  )
+  (fp_text value Converter_ACDC_Recom_RAC20-xxDK_THT (at 22.75 25.01) (layer F.Fab)
+    (effects (font (size 1 1) (thickness 0.15)))
+  )
+  (fp_line (start -2.5 -3.39) (end 49 -3.39) (layer F.Fab) (width 0.1))
+  (fp_line (start 49 -3.39) (end 49 24.01) (layer F.Fab) (width 0.1))
+  (fp_line (start 49 24.01) (end -3.5 24.01) (layer F.Fab) (width 0.1))
+  (fp_line (start -3.5 24.01) (end -3.5 -2.39) (layer F.Fab) (width 0.1))
+  (fp_line (start -3.5 -2.39) (end -2.5 -3.39) (layer F.Fab) (width 0.1))
+  (fp_line (start -4 -0.89) (end -4 -3.89) (layer F.SilkS) (width 0.12))
+  (fp_line (start -4 -3.89) (end -1 -3.89) (layer F.SilkS) (width 0.12))
+  (fp_line (start -3.62 -3.51) (end 49.12 -3.51) (layer F.SilkS) (width 0.12))
+  (fp_line (start 49.12 -3.51) (end 49.12 24.13) (layer F.SilkS) (width 0.12))
+  (fp_line (start -3.62 24.13) (end 49.12 24.13) (layer F.SilkS) (width 0.12))
+  (fp_line (start -3.62 -3.51) (end -3.62 24.13) (layer F.SilkS) (width 0.12))
+  (fp_line (start -3.75 -3.64) (end 49.25 -3.64) (layer F.CrtYd) (width 0.05))
+  (fp_line (start 49.25 -3.64) (end 49.25 24.26) (layer F.CrtYd) (width 0.05))
+  (fp_line (start -3.75 24.26) (end 49.25 24.26) (layer F.CrtYd) (width 0.05))
+  (fp_line (start -3.75 -3.64) (end -3.75 24.26) (layer F.CrtYd) (width 0.05))
+  (pad 1 thru_hole rect (at 0 0) (size 1.8 1.8) (drill 1.2) (layers *.Cu *.Mask))
+  (pad 2 thru_hole circle (at 0 20.32) (size 1.8 1.8) (drill 1.2) (layers *.Cu *.Mask))
+  (pad 3 thru_hole circle (at 45.72 20.32) (size 1.8 1.8) (drill 1.2) (layers *.Cu *.Mask))
+  (pad 4 thru_hole circle (at 45.72 10.16) (size 1.8 1.8) (drill 1.2) (layers *.Cu *.Mask))
+  (pad 5 thru_hole circle (at 45.72 0) (size 1.8 1.8) (drill 1.2) (layers *.Cu *.Mask))
+  (fp_text user %R (at 22.75 10.31) (layer F.Fab)
+    (effects (font (size 1 1) (thickness 0.15)))
+  )
+  (fp_text user "" (at -3 0) (layer F.SilkS)
+    (effects (font (size 1 1) (thickness 0.15)))
+  )
+  (model ${KISYS3DMOD}/Converter_DCDC.3dshapes/Converter_ACDC_Recom_RAC20-xxDK_THT.wrl
```

Comment:
Hi @evanshultz and @myfreescalewebpage , this 3D path is wrong, because the script used was about DC/DC converters, can AC/DC added at the scripts?
If not i will advice to manually edit..Same for the second part.
I haven't reviewed them yet..

Reply:
In my opinion it would be great to make the script more generic e.g. passing the library as a parameter. Also passing the drill diameter without any pad information would be nice as this dimensions can be calculated via the IPC standard and most datasheets don't specify the pads.